benchmark/req_bytes.go

```diff
@@ -31,5 +29,10 @@ func getRequestBytes(n int) []byte {
 }
 
 func getRequestString(n int) string {
-	return base64.RawStdEncoding.EncodeToString(getRequestBytes(n))[:n]
+	chars := []byte("ABCDEFGHIJKLMNOPQRSTUVWXYZabcedefghijklmnopqrstuvwxyz")
+	bs := make([]byte, n)
+	for i := range bs {
+		bs[i] = chars[i%len(chars)]
+	}
+	return string(bs)
```

Comment:
Can we leave a TODO to replace this with base64 encoding once we drop Go 1.4 support?